src/router/index.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable radix */
+
+'use strict';
+
+const express = require('express');
+const db = require('./../db');
+const products = require('./../db/products.json');
+
+const router = express.Router();
+
+// router.get('/', productsRoutes.allProducts);
+
+// creating a route - route gets all product
+router.get('/', (req, res) => res.json(products));
+
+
+// registerting a new user
+router.post('/register', (req, res, next) => {
+  console.log('New registeration in process');
+  db.register(req.body);
+  res.sendStatus(200);
+  next();
+});
+
+// logging in
+router.post('/login', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+//  get single product
+router.get('/db/products/:id', (req, res, next) => {
+  const found = products.some(product => product.id === console.log(parseInt(req.params.id)));
+
+  if (found) {
+    res.json(products.filter(product => product.id === parseInt(req.params.id)));
```

Comment:
This filter works but since you know the product exists in your database (found === true), you can just do:

```
if (found) {
    res.json(products[req.params.id])
```

Reply:
Alright i will try it out. Thank you for your suggestion.